teamworkApp/lib/overallBar.py

```diff
@@ -0,0 +1,50 @@
+from enum import IntEnum
+import sqlite3
+import matplotlib.pyplot as plt
+import numpy as np
+from findType import getType
+
+DB = 'db/development.sqlite3'
+Style = IntEnum('Style', 'Contributor, Collaborator, Communicator, Challenger', start=0)
+
+def getStylesData():
+    """Returns number of students per style"""
+
+    # Get all scores
+    conn = sqlite3.connect(DB)
+    c = conn.cursor()
+    contributor = c.execute('SELECT contributor FROM styles').fetchall()
+    collaborator = c.execute('SELECT collaborator FROM styles').fetchall()
+    communicator = c.execute('SELECT communicator FROM styles').fetchall()
+    challenger = c.execute('SELECT challenger FROM styles').fetchall()
```

Comment:
You should probably only do one database call such as: 'SELECT * FROM styles'. This should give you a list of tuples that is organized very similarly to the student scores list that you create during each for loop. It tends to be costly to make database calls so it is good to limit the number that you need to make.